hvac/api/secrets_engines/consul.py

```diff
@@ -37,7 +37,7 @@ def configure_access(self, address, token, scheme="http", mount_point=DEFAULT_MO
         return self._adapter.post(
             url=api_path,
             json=params,
-        ).json()
```

Comment:
Mind breaking out these commits to another PR? Since PR's are being used for auto-generated GitHub release notes / changlog entries at this point, it is nice to keep em logically distinct.